step10/get_json.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+)
+
+type person struct {
+	Name  string `json:"name"`
+	Age   int    `json:"age"`
+	Birth string `json:"birth"`
+}
```

Comment:
JSONだとどう出力されるのかコメントで示しておくと分かりやすいかも！？

```suggestion
/**
	{
		"name": "hoge",
		"age": 22,
		"birth": "2020年5月3日"
	}
*/
type person struct {
	Name  string `json:"name"`
	Age   int    `json:"age"`
	Birth string `json:"birth"`
}
```

Reply:
ありがとうございます！追記しましたー！